### ISSUE TITLE
CKEditor

### ISSUE THREAD

**Issue by mathiashelin** [vytenizs/forkcms#18]
CKEditor should definitely be updated, the current version isn't supported by IE11

**Comment by vytenizs** [comment]
Should i do this job? :D

**Comment by mathiashelin** [comment]
No, just posting stuff that should be done ;) When I find some time, I can take a look at this

**Comment by vytenizs** [comment]
Good :)

**Comment by vytenizs** [comment]
Why bower was not used to install ckeditor?

**Comment by mathiashelin** [comment]
Not sure if it's possible to use bower + custom plugins, but I'll look into it

**Comment by vytenizs** [comment]
I do not know that for certain, but i think using bower should be cool :D
